homeassistant/components/awair/__init__.py

```diff
@@ -21,13 +27,13 @@
     API_TIMEOUT,
     DOMAIN,
     LOGGER,
-    UPDATE_INTERVAL_CLOUD,
-    UPDATE_INTERVAL_LOCAL,
-    AwairResult,
 )
 
 PLATFORMS = [Platform.SENSOR]
 
+UPDATE_INTERVAL_CLOUD = timedelta(minutes=5)
+UPDATE_INTERVAL_LOCAL = timedelta(seconds=30)
```

Comment:
This seems to be pretty unneeded.

Personally, I even prefer what was originally done and don't think this is an improvement.

We don't have official guidelines on this either.

../Frenck